Formula/pcl.rb

```diff
@@ -67,6 +59,9 @@ def install
       "-DBUILD_apps_modeler:BOOL=OFF"
     end
 
+    # The AppleClang versions shipped on current MacOS versions do not support the -march=native flag on arm
+    args << "-DPCL_ENABLE_MARCHNATIVE:BOOL=OFF" if OS.mac? && (MacOS.version <= :ventura) && Hardware::CPU.arm?
```

Comment:
```suggestion
    # The AppleClang versions shipped on current MacOS versions do not support the -march=native flag on arm
    args << "-DPCL_ENABLE_MARCHNATIVE:BOOL=OFF" if build.bottle?
```
We want to avoid `-march=native` whenever building binaries for distribution.